smartapps/smartthings/hue-connect.src/hue-connect.groovy

```diff
@@ -816,7 +817,7 @@ private sendBasicEvents(device, param, value) {
 
 	switch (param) {
 		case "on":
-			device.sendEvent(name: "switch", value: (value == true) ? "on" : "off")
+			device.sendEvent(name: "switch", value: (value == true) ? "on" : "off", isStateChange: true)
```

Comment:
Yes, but forcing the isStateChange value will cause smartapps to fire repeatedly if they are listening to the on or off event on the bulb